manager-bundle/src/HttpKernel/ContaoCache.php

```diff
@@ -31,45 +31,40 @@ class ContaoCache extends HttpCache implements CacheInvalidation
     /**
      * @var bool
      */
-    private $wasBypassed;
+    private $isDebug;
 
     public function __construct(ContaoKernel $kernel, string $cacheDir = null)
     {
         parent::__construct($kernel, $cacheDir);
 
+        $this->isDebug = $kernel->isDebug();
+
+        $this->addSubscriber(new StripCookiesSubscriber(array_filter(explode(',', getenv('COOKIE_WHITELIST') ?: ''))));
```

Comment:
Is this meant to be `?:` or `??`?

Reply:
No `?:` is correct. `getenv()` returns `false` if it isn't set. `??` is short for `isset()` which makes no sense here.